examples/raw.rs

```diff
@@ -11,72 +11,106 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-extern crate futures;
-extern crate tikv_client;
-
+use futures::future::Future;
 use std::path::PathBuf;
+use tikv_client::{raw::Client, Config, Key, KvPair, Result, Value};
 
-use futures::future::Future;
-use tikv_client::*;
+const KEY: &str = "TiKV";
+const VALUE: &str = "Rust";
+const CUSTOM_CF: &str = "custom_cf";
 
-fn main() {
-    let config = Config::new(vec!["127.0.0.1:3379"]).with_security(
+fn main() -> Result<()> {
+    // Create a configuration to use for the example.
+    // Optionally encrypt the traffic.
+    let config = Config::new(vec![
+        "192.168.0.100:3379", // Avoid a single point of failure,
```

Comment:
can we use clap to pass all as command args? So we don't need to rebuild it when we want to communicate with another cluster.

Reply:
See #16 :) Let's leave that for another PR, this PR is already far too large.